Bookish.Web/Views/Home/Barcodes.cshtml

```diff
@@ -1,7 +1,17 @@
-﻿@{
+﻿@model BarcodeViewModel
+@{
     ViewData["Title"] = "Barcodes";
```

Comment:
The purpose of this page isn't just to display the barcodes I believe, it's also where the user gets directed once they've added a new book, right? In that case I'd make it a bit more like a confirmation page, with some text like "Book X has been added, here are barcodes for you to print" 🙂